tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go

```diff
@@ -0,0 +1,712 @@
+//go:build e2e
+// +build e2e
+
+package opentelemetry_metrics_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	prommodel "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/kedacore/keda/v2/pkg/metricscollector"
+	. "github.com/kedacore/keda/v2/tests/helper"
+)
+
+const (
+	testName          = "opentelemetry-metrics-test"
+	labelScaledObject = "scaledObject"
+	labelType         = "type"
+)
+
+var (
+	testNamespace             = fmt.Sprintf("%s-ns", testName)
+	deploymentName            = fmt.Sprintf("%s-deployment", testName)
+	monitoredDeploymentName   = fmt.Sprintf("%s-monitored", testName)
+	scaledObjectName          = fmt.Sprintf("%s-so", testName)
+	wrongScaledObjectName     = fmt.Sprintf("%s-wrong", testName)
+	wrongScalerName           = fmt.Sprintf("%s-wrong-scaler", testName)
+	cronScaledJobName         = fmt.Sprintf("%s-cron-sj", testName)
+	clientName                = fmt.Sprintf("%s-client", testName)
+	kedaOperatorPrometheusURL = "http://opentelemetry-collector.default.svc.cluster.local:8889/metrics"
```

Comment:
Could you rename this var? I thought that you were using the same endpoint as for promtheus tests.
Something like `otelBackendPrometheusURL`